icicle/include/icicle/fields/params_gen.h

```diff
@@ -114,12 +162,16 @@ namespace params_gen {
   static constexpr storage<limbs_count> m = params_gen::template get_m<limbs_count, 2 * modulus_bit_count>(modulus);   \
   static constexpr storage<limbs_count> montgomery_r =                                                                 \
     params_gen::template get_montgomery_constant<limbs_count, false>(modulus);                                         \
+  static constexpr storage<limbs_count> montgomery_r_sqr =                                                             \
+    params_gen::template get_montgomery_constant_sqr<limbs_count, false>(modulus);                                     \
   static constexpr storage<limbs_count> montgomery_r_inv =                                                             \
     params_gen::template get_montgomery_constant<limbs_count, true>(modulus);                                          \
+  static constexpr storage<limbs_count> mont_inv_modulus =                                                             \
+    params_gen::template get_montgomery_mult_constant<limbs_count>(modulus);                                           \
   static constexpr unsigned num_of_reductions =                                                                        \
     params_gen::template num_of_reductions<limbs_count, 2 * modulus_bit_count>(modulus, m);
 
-#define TWIDDLES(modulus, rou)                                                                                         \
+#define TWIDDLES(modulus)                                                                                              \
```

Comment:
what was this rou?

Reply:
root of unity, it's an old thing that should have been removed